### PR TITLE
fix: wrap the log printing in buildRoleLinks()

### PR DESCRIPTION
**Pull request by 00LT00**
Signed-off-by: 00LT00 <lightning@zerokirin.online>

Fix: https://github.com/casbin/casbin/issues/614

### CODE REVIEW

**Comment by hsluoyz** [comment]
@holayao hi, @00LT00 already made a PR. Please review.

File: model/policy.go [hunk 1]
```diff
@@ -43,6 +43,9 @@ func (model Model) BuildIncrementalRoleLinks(rm rbac.RoleManager, op PolicyOp, s
 
 // BuildRoleLinks initializes the roles in RBAC.
 func (model Model) BuildRoleLinks(rm rbac.RoleManager) error {
+	if log.GetLogger().IsEnabled() {
```
Comment (nodece) [inline model/policy.go:46]: I think you should add `log.GetLogger().IsEnabled()` to `PrintPolicy()`.

File: model/assertion.go [hunk 2]
```diff
@@ -83,6 +83,8 @@ func (ast *Assertion) buildRoleLinks(rm rbac.RoleManager) error {
 		}
 	}
 
-	log.LogPrint("Role links for: " + ast.Key)
+	if log.GetLogger().IsEnabled() {
```
Comment (nodece) [inline model/assertion.go:86]: unnecessary.

**Comment by holayao** [comment]
@nodece I was on vacation the other day, and I'm sorry I didn't review the code. I dont think this PR fixs the problem i am facing . The line which takes up a large heap is in this function  as [here](https://github.com/casbin/casbin/blob/d0d65d828c784211a47c23f9cc63b7429da0287e/model/assertion.go#L88). And i seems like this fuction does nothing but print log. So is it possible to check log flag before execute this function?  Thanks

**Comment by nodece** [comment]
@holayao We have added checks.
https://github.com/casbin/casbin/blob/d0d65d828c784211a47c23f9cc63b7429da0287e/rbac/default-role-manager/role_manager.go#L231

**Comment by holayao** [comment]
@nodece  Sorry, i didn't notice that .Thank you so much!

**Pull request merged**